ci/prow/e2e-incluster-build

```diff
@@ -31,7 +31,7 @@ echo "Wait for the job to be created..."
 timeout 1m bash -c 'until oc get job -l kmm.node.kubernetes.io/module.name | grep kmm; do sleep 1; done'
 
 echo "Check that the module gets loaded on the node..."
-timeout 1m bash -c 'until check_kmm_kmod; do sleep 3; done'
+timeout 1m bash -c 'until check_kmm_mod; do sleep 3; done'
```

Comment:
Can we extract `check_kmm_kmod` function to another file and use it? We are just duplicating it ATM.

Reply:
why? it is a one-line function and extract it to another file would imply to source it from the scripts, losing clarity IMHO